src/main/scala/org/constellation/domain/p2p/PeerHealthCheck.scala

```diff
@@ -17,22 +17,29 @@ import scala.concurrent.duration._
 
 class PeerHealthCheck[F[_]](cluster: Cluster[F], apiClient: ClientInterpreter[F], metrics: Metrics)(
   implicit F: Concurrent[F],
-  C: ContextShift[F],
-  T: Timer[F]
+  CS: ContextShift[F],
+  T: Timer[F],
+  C: Clock[F]
 ) {
   val logger: SelfAwareStructuredLogger[F] = Slf4jLogger.getLogger[F]
   val periodicCheckTimeout: FiniteDuration = 5.seconds
   val confirmationCheckTimeout: FiniteDuration = 5.seconds
   val ensureSleepTimeBetweenChecks: FiniteDuration = 3.seconds
   val ensureDefaultAttempts = 3
+  val cacheAfterChecks = ensureDefaultAttempts + 2
+  val cacheForTime = 30 * 1000 // 30 * 1000ms
```

Comment:
```scala
import scala.concurrent.duration._

val cacheForTime = 30.seconds // 🚀 
```

Reply:
yeah, I know about this. but clock is `Long` so I have milliseconds. then, would have to convert long into finite duration to compare it.

Reply:
Sure, but anyway. I would keep `val cacheForTime = 30.seconds` in declaration and then operate on it's `toMillis` when comparing as it is just more readable